index.bs

```diff
@@ -485,31 +485,31 @@ Issue: The above permission checks preclude {{Worker}} support as written.
 
 
 <!-- ============================================================ -->
-## The {{FontMetadata}} interface ## {#fontmetadata-interface}
+## The {{FontData}} interface ## {#fontdata-interface}
 <!-- ============================================================ -->
 
-A {{FontMetadata}} provides details about a font face. Each {{FontMetadata}} has an associated [=/font representation=].
+A {{FontData}} provides details about a font face. Each {{FontData}} has an associated [=/font representation=].
 
 <div class="domintro note">
 
-    : |metadata| . {{FontMetadata/postscriptName}}
+    : |fontdata| . {{FontData/postscriptName}}
     :: The PostScript name for the font. Example: "`Arial-Bold`".
 
-    : |metadata| . {{FontMetadata/fullName}}
+    : |fontdata| . {{FontData/fullName}}
     :: The full font name, including family subfamily names. Example: "`Arial Bold`"
 
-    : |metadata| . {{FontMetadata/family}}
+    : |fontdata| . {{FontData/family}}
     :: The font family name. This corresponds with the CSS 'font-family' property. Example: "`Arial`"
 
-    : |metadata| . {{FontMetadata/style}}
+    : |fontdata| . {{FontData/style}}
     :: The font style (or subfamily) name. Example: "`Regular`", "`Bold Italic`"
 
 </div>
 
 
 <xmp class=idl>
 [Exposed=Window]
-interface FontMetadata {
+interface FontData {
```

Comment:
This probably invites a question about how it interacts with the https://drafts.csswg.org/css-font-loading/#dom-fontface-data-slot slot in the CSSOM FontFace object … does it wind up being the case that you can pass the bytes from `.blob()` into [that constructor](https://drafts.csswg.org/css-font-loading/#fontface) and get an appropriate font object back?

Reply:
I believe so, yes. In an early iteration of this spec we were actually returning FontFace objects here but were pushed away from doing that (hopefully captured in the explainer)